src/transformers/feature_extraction_sequence_utils.py

```diff
@@ -107,6 +108,8 @@ def pad(
                   different lengths).
             max_length (:obj:`int`, `optional`):
                 Maximum length of the returned list and optionally padding length (see above).
+            truncation (:obj:`bool`, `optional`):
```

Comment:
Different to Tokenizers, we just need a True/False boolean here since we don't have input_pairs, ...

Reply:
Why is the default to None then? If it's just True/False, it should default to False and there is no need for a third state.